pkg/datasources/network_policies_acceptance_test.go

```diff
@@ -0,0 +1,200 @@
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.0.show_output.0.entries_in_blocked_ip_list", "0"),
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.0.show_output.0.entries_in_allowed_network_rules", "0"),
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.0.show_output.0.entries_in_blocked_network_rules", "0"),
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.0.show_output.0.comment", ""),
+
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.0.describe_output.#", "1"),
+					resource.TestCheckNoResourceAttr("data.snowflake_network_policies.test", "network_policies.0.describe_output.0.allowed_ip_list"),
+					resource.TestCheckNoResourceAttr("data.snowflake_network_policies.test", "network_policies.0.describe_output.0.blocked_ip_list"),
+					resource.TestCheckNoResourceAttr("data.snowflake_network_policies.test", "network_policies.0.describe_output.0.allowed_network_rule_list"),
+					resource.TestCheckNoResourceAttr("data.snowflake_network_policies.test", "network_policies.0.describe_output.0.blocked_network_rule_list"),
+				),
+			},
+			{
+				Config: networkPolicyConfigBasic(id.Name(), false),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.#", "1"),
+					resource.TestCheckResourceAttrSet("data.snowflake_network_policies.test", "network_policies.0.show_output.0.created_on"),
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.0.show_output.0.name", id.Name()),
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.0.show_output.0.entries_in_allowed_ip_list", "0"),
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.0.show_output.0.entries_in_blocked_ip_list", "0"),
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.0.show_output.0.entries_in_allowed_network_rules", "0"),
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.0.show_output.0.entries_in_blocked_network_rules", "0"),
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.0.show_output.0.comment", ""),
+
+					resource.TestCheckResourceAttr("data.snowflake_network_policies.test", "network_policies.0.describe_output.#", "0"),
+				),
+			},
+		},
+	})
+}
+
+func networkPolicyConfigBasic(name string, withDescribe bool) string {
+	return fmt.Sprintf(`
+	resource "snowflake_network_policy" "test" {
+		name = "%v"
+	}
+
+	data "snowflake_network_policies" "test" {
+		with_describe = %t
+		like = snowflake_network_policy.test.name
+	}
+`, name, withDescribe)
+}
+
+func networkPolicyConfigComplete(
```

Comment:
Why are these configs not in `testdata`?

Reply:
I think I have already answered in other PRs that we do not enforce one over the other, they are both not the greatest, so please let's not discuss it over and over again in the different PRs